packages/demo/src/App.svelte

```diff
@@ -114,38 +114,7 @@
       gnosis,
       dcent
     ],
-    chains: [
```

Comment:
@mahmud-bn Is there a reason we want to get rid of these chains?

Reply:
Oh! Fixed that. I was just re-checking the chain error.